libs/api/__tests__/api/private/index.ts

```diff
@@ -436,6 +436,11 @@ export type CreatedToken = {
   token: Scalars['String'];
 };
 
+export enum Currency {
+  Chf = 'CHF',
```

Comment:
Why this is camelcase? After fighting with enums in federation I would prefer all of them are same as values.
Is this only generator?

Reply:
This is generated and generated enums are always camelcase by the graphql generator